string job_id
string id
string destination
string item
string agent
string status